app/src/androidTest/java/edu/ucsb/cs56/projects/android/conversion_showdown/baseCalculatorTest.java

```diff
@@ -96,7 +96,7 @@ public void testCalcOctal() {
     @Test
     public void testCalcHex() {
         onView(withId(R.id.firstInput))
-                .perform(typeText("F"), closeSoftKeyboard());
+                .perform(typeText("f"), closeSoftKeyboard());
```

Comment:
Make sure the project supports both F and f for hexadecimal inputs.

Reply:
I was asking to add support for both small and capital letters for hex value.

Reply:
Fixed